src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/arcadeAvailability.dashboard.json

```diff
@@ -902,9 +947,10 @@
       "linewidth": 4,
       "nullPointMode": "null",
       "options": {
-        "dataLinks": []
+        "alertThreshold": true
       },
       "percentage": false,
+      "pluginVersion": "7.4.3",
```

Comment:
Are these numbers right? I thought we rolled everything back to v6?

Reply:
Hadn't pushed new changes since the rollback so it has the old versions